clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -49,6 +49,8 @@ class StorageType(Enum):
     HDFS = "hdfs"
     LOCAL = "file"
     GCS = "gs"
+    S3A = "s3a"
+    S3 = "s3"
```

Comment:
Why do we add two, "s3a" and "s3"?

Reply:
I have the same question, because we only use the s3a scheme in the `S3FileSystemProvider`(https://github.com/apache/gravitino/blob/main/bundles/aws-bundle/src/main/java/org/apache/gravitino/s3/fs/S3FileSystemProvider.java#L44), is there any case will use the s3 scheme?